apis/v1alpha1/gatewayclass_types.go

```diff
@@ -98,18 +98,15 @@ type GatewayClassSpec struct {
 	// +kubebuilder:default={onlySameNamespace:true}
 	AllowedRouteNamespaces RouteNamespaces `json:"allowedRouteNamespaces,omitempty"`
 
-	// ParametersRef is a controller-specific resource containing
-	// the configuration parameters corresponding to this
-	// class. This is optional if the controller does not require
-	// any additional configuration.
-	//
-	// Valid resources for reference are up to the Controller. Examples
-	// include "configmaps" (omit or specify the empty string for the group
-	// to indicate the core API group) or a custom resource (CRD).  Omitting
-	// or specifying the empty string for both the resource and group
-	// indicates that the resource is "configmaps".  If the referent cannot
-	// be found, the GatewayClass's "InvalidParameters" status condition
-	// will be true.
+	// ParametersRef is a controller-specific resource containing the
+	// configuration parameters corresponding to this class. This is optional if
+	// the controller does not require any additional configuration.
+	//
+	// Parameters resources will be implementation specific custom resources.
+	// These resources will also be cluster-scoped.
```

Comment:
Can we use a more stronger language here? Like "These resources must be cluster-scoped"?

Reply:
Updated, thanks!